cortix/examples/__init__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the Cortix toolkit environment
+# https://cortix.org
+
+from cortix.examples.ideal_gas.particle_plotting import Particle_Plot
```

Comment:
Please remove these lines as they cause the ideal gas simulation to be automatically imported which is problematic because shapely is not a dependency.